src/pybind/kaldi_pybind.cc

```diff
@@ -44,6 +44,19 @@ void sequential_matrix_reader(py::module &m, std::string suffix) {
         .def("Close", &SequentialTableReader<Holder>::Close);  
 }
 
+template<class Holder>
+void random_access_matrix_reader(py::module &m, std::string suffix) {
+    std::string name = "RandomAccessBaseFloatMatrixReader" + suffix;
```

Comment:
I don't think the names RandomAccessBaseFloatMatrixReader_Matrix and RandomAccessBaseFloatMatrixReader_Vector make sense.  It's either a matrix reader or a vector reader, not both.